### ISSUE TITLE
Sometimes gulp del cause EPERM error

### ISSUE THREAD

**Issue by havenchyk** [gulpjs/gulp#738]
Hi. 

I'm trying del in my `clean` task.

```
gulp.task('clean', function(cb) {
 del('build/**', cb);
});

gulp.task('build', ['clean'], function() {
    ...
});
```

And from time to time during running `build` task I've got 

> 'clean' errored after 24 ms
>  Error: EPERM, rmdir 'D:\dev\projects...

OS: Windows 7.

Is it problem of my implementation or just wrong behavior on Windows?

Sorry if it's not a good place to post this problem.

Thanks

**Comment by rmorse** [comment]
I'm not sure if it's related directly to your problem but I was also getting this  error intermittently - although checking my files & folders, the task had successfully run.

Looking at your particular use case it doesn't seem like this applies but I hope this helps others:

First I was also getting another issue with `watch` on my build picking up newly created files:
https://github.com/floatdrop/gulp-watch/issues/1

After finding [this solution](http://stackoverflow.com/questions/22391527/gulps-gulp-watch-not-triggered-for-new-or-deleted-files/26851844#26851844) and solving the problem above, my `del` issue disappeared too: 

Basically when my globs started with `./` it caused this error - simply remove it so:

`./scripts/**`

would become:

`scripts/**`

**Comment by egor-rock** [comment]
Guys, I ran into the same problem, and nothing helped. I also use syncing in Google Drive, but I tried it in a separate folder. In my case, everything turned out to be quite simple - this is the work of an antivirus. I created a new folder, added it to the exceptions and moved the project files there. After that, everything works as it should! Thank you havenchyk! And to all of you! You are the best!!
